contracts/RestrictedNORI.sol

```diff
@@ -885,7 +882,7 @@ contract RestrictedNORI is
       scheduleId,
       quantityToRevoke
     );
-    // slither-disable-next-line calls-loop
+    // slither-disable-next-line calls-loop no option for batch send on ERC777
```

Comment:
While true for ERC777s, we wrote this function ourselves so this isn't  true here

https://github.com/nori-dot-eco/contracts/blob/aa514e502ee4623ab2ba2a57f4d927789dedf595/contracts/ERC777PresetPausablePermissioned.sol#L55

Reply:
ah I see... so the reason slither is giving us this error is actually because `_revokeUnreleasedTokens` is inside a loop inside `batchRevokeUnreleasedTokens`.  I was thinking maybe we could put a `batchSend` at the end of `batchRevokeUnreleasedTokens` that handles all the sends at once, but as things are currently written, we need the singular `_revokeUnreleasedTokens` to calculate the actual quantity to revoke for each schedule in the batch.  One path forward would be to either have `_revokeUnreleasedTokens` return the quantity to revoke for each schedule, and then we construct an array and use it in the batch send.  Or to expand that function entirely inside of `batchRevokeUnreleasedTokens` and don't bother with the private implementation.  will think on this and leave a todo at the very least

Reply:
the todo that's now on `batchRevokeUnreleasedTokens` basically addresses this - When we optimize that function to take only a single project id of removals this situation will resolve itself